lib/YuiRestClient.pm

```diff
@@ -65,7 +70,7 @@ sub setup_libyui_running_system {
     my $host = get_var('YUI_SERVER');
     record_info('PORT',   "Used port for libyui: $port");
     record_info('SERVER', "Connecting to: $host");
-    set_var('YUI_PARAMS', "YUI_HTTP_PORT=$port YUI_HTTP_REMOTE=1 YUI_REUSE_PORT=1");
+    set_var('YUI_PARAMS', get_yui_params_string());
```

Comment:
could we pass the port instead of getting the variable twice?
Also we could use it below in `sub set_libyui_backend_vars`

Reply:
Makes sense, I will use `get_required_var` here than, as it should be set by this point of time.